### PR TITLE
add ansible-galaxy cli documentation for parallel execution

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
Copy-pasta from #79387 to document the same for ansible-galaxy.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ansible-galaxy

### CODE REVIEW

**Review by bcoca — APPROVED** [review]

**Review by sivel — APPROVED** [review]

File: lib/ansible/cli/galaxy.py [hunk 1]
```diff
@@ -197,7 +197,10 @@ def api(self):
 
 
 class GalaxyCLI(CLI):
-    '''command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.'''
+    '''Command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.
```
Comment (webknjaz) [inline lib/ansible/cli/galaxy.py]: The "roles" bit no longer seems accurate since it manages collections too. Though I suppose fixing that may be fine in a separate PR.
Additionally, it'd be nice to follow PEP 257, having a short title, an empty line and a multi-line description with full sentences.

Comment (s-hertel) [reply lib/ansible/cli/galaxy.py]: Heh, I didn't even notice it only mentions roles. Good catch.

File: lib/ansible/cli/galaxy.py [hunk 2]
```diff
@@ -197,7 +197,10 @@ def api(self):
...
+       None of the CLI tools are designed to run concurrently with themselves,
+       you should use an external scheduler and/or locking to ensure there are no clashing operations.
```
Comment (s-hertel) [inline lib/ansible/cli/galaxy.py]:
```suggestion
    '''Command to manage Ansible roles and collections.

       None of the CLI tools are designed to run concurrently with themselves,
       you should use an external scheduler and/or locking to ensure there are no clashing operations.
```

File: lib/ansible/cli/galaxy.py (hunk 1 above)
Comment (s-hertel) [reply lib/ansible/cli/galaxy.py]: Fixed the formatting, made the title simpler. Feel free to suggest a better description.

File: lib/ansible/cli/galaxy.py [hunk 3]
```diff
@@ -197,7 +197,11 @@ def api(self):
 
 
 class GalaxyCLI(CLI):
-    '''command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.'''
+    '''Command to manage Ansible roles and collections.
+
+       None of the CLI tools are designed to run concurrently with themselves,
+       you should use an external scheduler and/or locking to ensure there are no clashing operations.
```
Comment (webknjaz) [inline lib/ansible/cli/galaxy.py]: nit: I'd make these separate sentences. Short phrases are easier to consume for the reader.
```suggestion
       None of the CLI tools are designed to run concurrently with themselves.
       Use an external scheduler and/or locking to ensure there are no clashing operations.
```

**Review by webknjaz — APPROVED** [review]

File: lib/ansible/cli/galaxy.py (hunk 3 above)
Comment (s-hertel) [reply lib/ansible/cli/galaxy.py]: Fixed. I had just copy-pasted what the docs team had signed off on for `ansible-pull`.
